File: src/main/java_backup/com/example/liferay/reactpoc/portlet/MainPortletConfiguration.java
package com.example.liferay.reactpoc.portlet;

import aQute.bnd.annotation.metatype.Meta;

@Meta.OCD(id = "MainPortletConfiguration")
public interface MainPortletConfiguration {
    @Meta.AD(
        deflt = "false",
        required = false
    )
    public String universalRender();

    @Meta.AD(
        deflt = "false",
        required = false
    )
    public String historyApi();
}

File: src/main/kotlin/com/example/liferay/reactpoc/portlet/MainPortletConfiguration.kt
package com.example.liferay.reactpoc.portlet

import aQute.bnd.annotation.metatype.Meta

@Meta.OCD(id = "MainPortletConfiguration")
interface MainPortletConfiguration {
    @Meta.AD(deflt = "false", required = false)
    fun universalRender(): String

    @Meta.AD(deflt = "false", required = false)
    fun historyApi(): String
}
